src/HabitTracker.Domain/Helpers/ResponsePackage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HabitTracker.Domain.Helpers;
+public class ResponsePackage
+{
+    #region Constructor
+    public ResponsePackage()
```

Comment:
See code style throughout the solution.
There should be whitespace between the regions and methods.
But not between regions.